code/modules/mob/living/simple_animal/ascendant_shadowling.dm

```diff
@@ -0,0 +1,39 @@
+/mob/living/simple_animal/ascendant_shadowling
+	name = "???"
+	desc = "A floating eldritch horror. Its eyes are terribly aware. You should probably run."
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "shadowling_ascended"
+	icon_living = "shadowling_ascended"
+	speak_emote = list("telepathically chitters", "telepathically chirps", "telepathically booms")
+	force_threshold = INFINITY //Can't die by normal means
+	health = 100000
+	maxHealth = 100000
```

Comment:
I think the speak_emotes should be more menacing.

"thunders", "booms", "shrieks", "howls" , "roars" ,"screams",